lib/replicator.js

```diff
@@ -2218,3 +2240,7 @@ function onwireextension (m, c) {
 function setDownloadingLater (repl, downloading, session) {
   repl.setDownloadingNow(downloading, session)
 }
+
+function isBlockRequest (req) {
+  return req !== null && req.block !== null
```

Comment:
Won't `req.block` be undefined if it's not a block request, since it's either set or not set? (in which case `req.block != null` is better)

Reply:
no its always explicit (i hate undefined haha)